Source/WebCore/css/CSSCounterStyleDescriptors.h

```diff
@@ -59,7 +59,7 @@ struct CSSCounterStyleDescriptors {
     };
     struct Pad {
         unsigned m_padMinimumLength = 0;
-        Symbol m_padSymbol = "-"_s;
+        Symbol m_padSymbol;
```

Comment:
ditto for this file

Reply:
Since I was here I made a tiny initialization adjustment.